content/en/blog/_posts/2022-09-05-pod-os-field/index.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: blog
+title: "Identifying Pod Operating System Authoritatively During Pod Admission Time"
+date: 2022-09-20
+slug: pod-os-field-explained
+---
+
+**Authors:** Ravi Gudimetla (Apple)
+
+This article describes how to identify pod's operating system authoritatively using the `os` field in the pod spec, and how that
+helps in securing Kubernetes clusters while ensuring that the workloads can run smoothly.
+
+## What problems does this solve?
+
+In the first releases of Kubernetes the `kube-apiserver` did not record any detail about which operating system a Pod should run on. Initially, Kubernetes only supported Linux
+nodes; later, Kubernetes added support for Windows nodes (stable since Kubernetes v1.14).
+You could use [labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/)
```

Comment:
Maybe we should just say there are many different ways to give the scheduler a hint?
One of the more common ways is to use a nodeSelector.

Reply:
I'll reword it to `While Kubernetes has multiple ways to hint scheduler about the node on which pod needs to run, a more common way is to use nodeSelector`, WDYT?